### PR TITLE
feat: nuxt3

### PR DESCRIPTION
**Pull request by farnabaz**
Rewrite module for Nuxt 3

### CODE REVIEW

File: src/runtime/components/component.vue [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<template>
+  <svg :view-box="viewBox" :class="icon.class">
```
Comment (or2e) [inline src/runtime/components/component.vue:2]: `:view-box="viewBox"` -> `:viewBox="viewBox"`

Comment (bmulholland) [reply src/runtime/components/component.vue:2]: The code is correct: https://v2.vuejs.org/v2/style-guide/?redirect=true#Prop-name-casing-strongly-recommended

Comment (or2e) [reply src/runtime/components/component.vue:2]: Does not work in Vue version 3 :(

**Comment by igamov** [comment]
I installed your branch nuxt3 and got an error: Cannot find module @nuxtjs\svg-sprite\dist\module.cjs

**Comment by Botz** [comment]
Are there any updates on this?

**Pull request merged**